components/x-audio/src/components/ClickableContainer.jsx

```diff
@@ -0,0 +1,28 @@
+import { h } from '@financial-times/x-engine';
+import * as PropTypes from 'prop-types';
+
+const CLICKABLE_ELEMENTS = ['a', 'button', 'submit', 'input']
+
+export const ClickableContainer = ({
+	children,
+	onClick,
+	...props
+}) => (
+	<div
+		{ ...props }
+		role='button'
+		tabIndex='0'
+		onKeyPress={() => onClick()}
```

Comment:
Is this to deal with a particular scenario?

Reply:
eslint caught it: https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/click-events-have-key-events.md

Not sure if it's best to restrict it to a particular key? /cc @debugwand

Reply:
the advice at the start of that plugin is wrong. and he doesn't mention anything about focus state.

certainly on desktop, click events are also triggered by pressing enter or space when you are focussed on an element. I believe click events are also triggered by double tapping when you are running a screenreader on a touch device. I am going to double check that so I'm not giving you bad advice.

Reply:
tested with https://jsbin.com/weginoduka/edit?html,js,output on webviewtest app and screenreader and it works as expected.

I don't know how screenreaders will interpret role of button on a div that contains other stuff, so when its merged in we can test with the phone and find out. It might need some hidden instructions to explain you get more controls when you activate it.

Reply:
ah good idea, will have a play when it's merged

Reply:
the one case I've found where it won't is using a keyboard and a non-typical element (as div here, anchors work) but no screenreader is active

In that case the keypresses we want to listen for are ones with keyCode 32 or 13
```
onKeyPress{(e) => {
if e.keyCode ===32 || e.keyCode===13 {
onClick();
}
});
```

Reply:
so: anchor element, button elements and checkbox inputs are the ones that don't need extra keypress handling. other elements like divs do, but only if you are not using a screenreader.